lua/ulx_yeet/sh_yeet.lua

```diff
@@ -186,13 +194,24 @@ local function playerDrop( ply, ent )
             if not IsValid( ragdoll ) then
                 hook.Remove( "Tick", hookName )
                 ent:Spawn()
+                ent.yeetRagdoll = nil
                 return
             end
-            if ragdoll:GetVelocity():Length() > 10 or ragdoll.cooldown > CurTime() then return end
+
+            if not IsValid( ragdoll.player ) then
+                ragdoll:Remove()
+                hook.Remove( "Tick", hookName )
+            end
+
+            if ragdoll:GetVelocity():Length() > 0.5 or ragdoll.cooldown > CurTime() then return end
             unRagdollPlayer( ragdoll )
             hook.Remove( "Tick", hookName )
         end)
     end)
 end
 
 hook.Add( "PhysgunDrop", "ulxPlayerDrop", playerDrop )
+
+hook.Add( "CanPlayerSuicide", "ulxYeetCanSuicideCheck", function( ply )
+    return not ply.yeetRagdoll
```

Comment:
We don't want to always return on this hook
```suggestion
    if ply.yeetRagdoll then return false end
```